### PR TITLE
updated generation of config to use fewer env variables

### LINKED ISSUE #26: Streamline environment variables
Instead of specifying ports for the urls in a separate variable, we should just use one variable, essentially dropping CLIENT_PORT and CONTINUUM_LOGIN_PORT for config.ts generation

* change API_HOST generation to use a CLIENT_API_URL only
* change LOGIN_URL generation to use a CONTINUUM_LOGIN_URL only

### PR DESCRIPTION
**Pull request by erezmus**
closes #26

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ const webpack = require('webpack');
 module.exports = {
     plugins: [
         new webpack.DefinePlugin({
-            API_HOST: JSON.stringify(`${process.env.CLIENT_API_URL}:${process.env.CLIENT_PORT}`),
-            LOGIN_URL: JSON.stringify(`${process.env.CONTINUUM_LOGIN_URL}:${process.env.CONTINUUM_LOGIN_PORT}/submit`),
+            API_HOST: JSON.stringify(`${process.env.CLIENT_API_URL}`),
```
Comment (diversemix) [inline .storybook/webpack.config.js]: Why the `JSON.stringify()` ?

Comment (erezmus) [reply .storybook/webpack.config.js]: otherwise we would havesay:
`API_HOST: http://localhost:3000`

instead of 
`API_HOST: "http://localhost:3000"`

Comment (diversemix) [reply .storybook/webpack.config.js]: Ah you mean its for the string quotation? E.g.
```
> JSON.stringify(process.env.EDITOR)
'"vim"'
> process.env.EDITOR
'vim'
```

Comment (erezmus) [reply .storybook/webpack.config.js]: yup

**Pull request merged**